src/Microsoft.SymbolStore/KeyGenerators/PEFileKeyGenerator.cs

```diff
@@ -13,15 +13,12 @@ namespace Microsoft.SymbolStore.KeyGenerators
     public class PEFileKeyGenerator : KeyGenerator
     {
         private const string CoreClrFileName = "coreclr.dll";
+        private const string ClrFileName = "clr.dll";
```

Comment:
It's odd that `ClrFileName` has ".dll" and `SosFileName` does not.

Reply:
Fixed, all of the filenames will have the extension. I was originally conflicted becaus some of the constants were just the prefixes, but now the long name file generator will extract the filename without the extension to construct the long name file variant.